ripple/src/main/webapp/main.html

```diff
@@ -25,14 +25,13 @@
       var auth = firebase.auth();
       var db = firebase.firestore();
     </script>
-    <!-- This needs to be after firebase initialization -->
-    <script src="auth.js"></script>
-    <script src="firestore.js"></script>
     <!-- ./Firebase configuration and initialization -->
     <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBo4gWG5T9oBscp7tHWzSL9L4masICev5U&region=US&libraries=places"></script>
     <script src="script.js"></script>
+    <script src="auth.js"></script>
     <script src="landing.js"></script>
     <script src="businesscarousels.js"></script>
+    <script src="trie.js"></script>
```

Comment:
Sarah and I were working on the autocomplete feature this evening but didn't end up finishing so I didn't include the file! Should I remove the script for now?

Reply:
Ideally when you submit code, it should be in a state so that if somebody else pulls the new code without any local changes it should not be broken. I would remove this line until trie.js is ready to be submitted.

Reply:
Got it, will remove!